src/components/profile/VerifyEditCode.js

```diff
@@ -38,23 +43,67 @@ const VerifyEditCode = props => {
     content,
   })
 
+  const reCaptchaRef = useRef()
```

Comment:
i've actually tested it and it doesnt seem to work. probably because we return the Captcha component as part of the hook, so trying to call launchCaptcha again after a failure doesnt work.
While it does work in the way i've implemented it.
After trying i've realized there are no gains here, the only thing you are saving is the "ref" that you dont need to define. but other wise there is 0 code saved and more importantly 0 optimization. another example of over trying to refactor everything without achieving anything substantial

Reply:
let's keep working implementation of course